lib/core/src/lib/form/components/widgets/core/form-field-validator.ts

```diff
@@ -51,22 +50,6 @@ export class RequiredFieldValidator implements FormFieldValidator {
 
     validate(field: FormFieldModel): boolean {
         if (this.isSupported(field) && field.isVisible) {
-            if (field.type === FormFieldTypes.DROPDOWN) {
-                if (field.hasMultipleValues) {
```

Comment:
just a note, please make sure we still have a correct behaviour of form validity as a whole, like disable 'Complete' button where `field.hasMultipleValues` and `Array.isArray(field.value) && !!field.value.length` return false